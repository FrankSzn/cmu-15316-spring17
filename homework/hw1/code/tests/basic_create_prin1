as principal admin password "admin" do
   create principal alice "alices_password"   
   return "success"
***