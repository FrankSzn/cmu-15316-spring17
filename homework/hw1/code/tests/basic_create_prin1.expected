{"status":"CREATE_PRINCIPAL"}
{"output":"success","status":"RETURNING"}
